stdlib/map.ml

```diff
@@ -59,6 +59,7 @@ module type S =
     val map: ('a -> 'b) -> 'a t -> 'b t
     val mapi: (key -> 'a -> 'b) -> 'a t -> 'b t
     val to_seq : 'a t -> (key * 'a) Seq.t
+    val to_seq_rev : 'a t -> (key * 'a) Seq.t
```

Comment:
There are a few functions in the `List` module whose names begin with `rev`, such as `rev_map`, and `rev_append`, but none whose names end with `rev` (except `rev`, I suppose).

I think it would be more consistent with existing conventions to call this `rev_to_seq`

Reply:
It makes sense. Done.